testing/visual-testing/tests/chip/basic/index.html

```diff
@@ -13,6 +13,11 @@
       content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0"
     />
     <title>Stencil Component Starter</title>
+    <style>
+      .chip-left-margin {
```

Comment:
Can we not simplify the selector? This way we won't have to add classes to all elements (also applies to other test files).
```suggestion
      ix-chip {
```

Reply:
Updated the code